docs/maps/search.asciidoc

```diff
@@ -13,13 +14,49 @@ You can create a layer that requests data from {es} from the following:
 
 ** Grid aggregation source
 
-** <<terms-join>>
+** <<terms-join>>. The search context is applied to both the terms join and the vector source when the vector source is provided by Elasticsearch documents.
 
 * <<heatmap-layer>> with Grid aggregation source
 
 [role="screenshot"]
 image::maps/images/global_search_bar.png[]
 
+[role="xpack"]
+[[maps-create-filter-from-map]]
+=== Creating filters from your map
+
+You can create filters by interacting with your map.
+Filters are applied to all layers in your map and other panels when your map is <<maps-embedding, embedded in a dashboard>>
+
```

Comment:
How about adding something like this:

*Elastic Maps* supports two types of interactive filters:

- Spatial filters
- Phrase filters

and link down the page.